docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md

```diff
@@ -429,6 +433,52 @@ The value of `core_pattern` can be a literal path or it can contain a pipe symbo
 
 - If the value of `core_pattern` contains a `|` pipe symbol (for example, `|/usr/share/apport/apport -p%p -s%s -c%c -d%d -P%P -u%u -g%g -- %E`), the core dump is being redirected to a specific collector on the underlying Kubernetes node, with the location depending on the exact collector. In this case, it is your responsibility to identify the location to which these files are written and retrieve them.
 
+### Use debug hooks with YugabyteDB in Kubernetes
+
+You can add your own commands to pre and post debug hooks to troubleshoot crashing YB-Master or YB-TServer pods. These commands are run before the database process starts and after the database process terminates or crashes.
+
+For example, to modify the debug hooks of a YB-Master, you will run following commands:
+
+```sh
+kubectl edit configmap -n <namespace> ybuni1-asia-south1-a-lbrl-master-hooks
+```
+
+This will open the configmap YAML in your editor.
+
+To add multiple commands to pre hook of `yb-master-0`, you can modify the `yb-master-0-pre_debug_hook.sh` key as follows:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ybuni1-asia-south1-a-lbrl-master-hooks
+data:
+  yb-master-0-post_debug_hook.sh: 'echo ''hello-from-post'' '
+  yb-master-0-pre_debug_hook.sh: |
+    echo "Running the pre hook"
+    du -sh /mnt/disk0/yb-data/
+    sleep 5m
+    # other commands here…
+  yb-master-1-post_debug_hook.sh: 'echo ''hello-from-post'' '
+  yb-master-1-pre_debug_hook.sh: 'echo ''hello-from-pre'' '
+```
+
+Once you save the file, the updated commands will be executed on the next restart of `yb-master-0`.
+
+You can run the following command to check the output of your debug hook:
+
+```sh
+kubectl logs -n <namespace> ybuni1-asia-south1-a-lbrl-yb-master-0 -c yb-master
+```
+
+Expect an output similar to the following:
+
+```output
+…
```

Comment:
...
use 3 dots instead of an ellipsis character

Reply:
Okay I will. Just curios: what is the reason to not use ellipsis?

Reply:
I thought it was causing this annoying red highlighting when reviewing, but it's still there with three dots so I'm not sure what the deal is. They both render the same on the actual site.